modxcms/revolution
# Update NodeJS up to latest LTS release

### What does it do?
Updates NodeJS up to version `14.15.0` and `node-sass` up to `5.0.0`, nothing else.

### Why is it needed?
To get possible to build front on the fresh nodejs version, which updates frequently.

### How to test
Have prepared on your local machine nodejs with version 14.15.0 or higher and go to _build/templates/default and run several commands:

```
rm -rf node_modules
npm install
grunt build
```

Build should be done without any **errors** and new compiled assets should be generated. However, several warnings still there because the old version of Bourbon is used, but updating to the new version require a lot of work under styles, that already was done for 3.x and does not make sense for 2.x.

### Related issue(s)/PR(s)
Almost the same thing for 3.x #15317
